bin/sa-can-read-s3.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+# I would like to demonstrate access to AWS resource (e.g. S3 bucket) from a pod. Idea:
+# create a bucket, put two objects in it (one public, one private), then … I suppose I
+# need to create a role with access to the private object, associate the role to a service
+# account in k8s, find an image with the aws CLI (or some s3 client) in it … ?
+
+set -euo pipefail
+
+emit_describe_cluster_policy() {
+    # Not used right now, but this permission is required in order to run `aws eks update-kubeconfig`:
+    echo '{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "eks:DescribeCluster"
+            ],
+            "Resource": "'"arn:aws:eks:${REGION}:${ACCOUNT_ID}:cluster/${CLUSTER_NAME}"'",
+            "Effect": "Allow"
+        }
+    ]
+}'
+}
+
+create_describe_cluster_policy() {
+    local role_name="can-describe-cluster"
+    aws iam create-policy \
+        --policy-name can-describe-cluster \
+        --description "Policy allowing to describe ${CLUSTER_NAME}" \
+        --policy-document "$(emit_describe_cluster_policy)"
+
+    aws iam attach-user-policy --role-name "${role_name}" --policy-arn "${S3_POLICY_ARN}"
```

Comment:
:warning: This function would need to be adapted in order to actually work -- also, it should be `--user-name`, not `--role-name`.